terraform-modules/k8s/main.tf

```diff
@@ -0,0 +1,33 @@
+# Pass variables through to nested modules.
```

Comment:
Ergh... I wanted this to allow for backwards-compat, but using this branch is causing a full cluster rebuild because all TF sees are the name changes from the resources being renamed / re-scoped under the module. Should we just leave the existing `k8s` module untouched?

Reply:
Thoughts on leaving the module in the k8s folder and just rename k8s-cluster.tf => k8-master.tf and make k8s-node-pool.tf a dynamic resource pending `node_count`  being greater than 0 ?

Reply:
Recording what we talked about in-person: Having separate modules will be useful for us because:
1. We want to track master and node pool definitions in separate resources (masters will be in our "core environments" repo, and node pools will be in per-project repos)
2. We've had use cases where we want to run multiple node pools per master, with different machine types (i.e. Kafka brokers on mem-optimized nodes, and agents on CPU-optimize nodes)